crates/uv-distribution/src/workspace.rs

```diff
@@ -901,10 +959,17 @@ impl VirtualProject {
     ///
     /// Similar to calling [`ProjectWorkspace::discover`] with a fallback to [`Workspace::discover`],
     /// but avoids rereading the `pyproject.toml` (and relying on error-handling as control flow).
+    ///
+    /// This method always uses absolute path, i.e. this method only supports discovering the main
+    /// workspace.
```

Comment:
I would clarify here that the method _requires_ that the `path` is absolute, or else it panics. I think the current phrasing could imply that the given path is turned _into_ an absolute path by this function if it isn't already.